cartoframes/io/carto.py

```diff
@@ -114,6 +119,17 @@ def to_carto(dataframe, table_name, credentials=None, if_exists='fail', geom_col
 
     context_manager = ContextManager(credentials)
 
+    if not ignore_quota_warning:
```

Comment:
We could extract here `me_data = context_manager.credentials.me_data` to have a bit more readable code.